bvm/ballerina-core/src/main/java/org/ballerinalang/model/TableJSONDataSource.java

```diff
@@ -39,19 +39,19 @@
  *
```

Comment:
Change "implementation for DataTable" to "implementation for Table"

Reply:
Fixed with 9f1d08c140b224ec932c0a0933457da9d3f401bf